hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/handlers/VolumeEntityHandler.java

```diff
@@ -117,21 +116,17 @@ public DUResponse getDuResponse(
   public QuotaUsageResponse getQuotaResponse()
           throws IOException {
     QuotaUsageResponse quotaUsageResponse = new QuotaUsageResponse();
-    String[] names = getNames();
-    List<OmBucketInfo> buckets = listBucketsUnderVolume(names[0]);
-    String volKey = getOmMetadataManager().getVolumeKey(names[0]);
-    OmVolumeArgs volumeArgs =
-            getOmMetadataManager().getVolumeTable().getSkipCache(volKey);
-    long quotaInBytes = volumeArgs.getQuotaInBytes();
-    long quotaUsedInBytes = 0L;
+    RootEntityHandler rootEntityHandler = new RootEntityHandler(
+        getReconNamespaceSummaryManager(),
+        getOmMetadataManager(),
+        getReconSCM(),
+        "/"
+    );
+    QuotaUsageResponse rootQuotaUsageResponse = 
+        rootEntityHandler.getQuotaResponse();
```

Comment:
in HDFS, -df returns capacity of the whole file system when df towards a directory, which we'll do similar here in Ozone.